libs/pilight/protocols/433.92/tcm.c

```diff
@@ -92,10 +92,7 @@ static void parseCode(void) {
 
 	humidity = binToDecRev(binary, 16, 23);
 
-	temp = binToDecRev(binary, 24, 35);
-	if(binary[24]) {
-		temp -= 4096;
-	}
+	temp = binToSignedRev(binary, 24, 35);
```

Comment:
Is defining `temp` and then assigning it to `temperature` just a left over from the previous code? Otherwise it could be removed.